modules/publicize/publicize-jetpack.php

```diff
@@ -447,7 +447,11 @@ function test_connection( $service_name, $connection ) {
 	 * Save a flag locally to indicate that this post has already been Publicized via the selected
 	 * connections.
 	 */
-	function save_publicized( $post_ID, $post, $update ) {
+	function save_publicized( $post_ID = null, $post = null, $update = null ) {
```

Comment:
> What if a plugin comes along and does something that we don't expect.

I'd also be interested to see if we're unexpectedly breaking by checking for null in all 3 cases.